### PR TITLE
Fixed small data discrepancy between Xee and pure computePixels().

### PR DESCRIPTION
**Pull request by dabhicusp**
By incorporating the following code into the main branch, i successfully addressed and resolved issue https://github.com/google/Xee/issues/40.(The mentioned internal test case was passed when utilizing the current solution.)

### CODE REVIEW

**Review by mahrsee1997 — APPROVED** [review]
LGTM.

File: xee/ext.py [hunk 1]
```diff
@@ -637,8 +637,8 @@ def __init__(self, variable_name: str, ee_store: EarthEngineStore):
 
     x_min, y_min, x_max, y_max = self.bounds
 
-    x_size = int(np.ceil((x_max - x_min) / np.abs(self.store.scale_x)))
-    y_size = int(np.ceil((y_max - y_min) / np.abs(self.store.scale_y)))
+    x_size = int(np.round((x_max - x_min) / np.abs(self.store.scale_x)))
+    y_size = int(np.round((y_max - y_min) / np.abs(self.store.scale_y)))
```
Comment (alxmrs) [inline xee/ext.py:641]: Did Sai’s modulus trick not work?

I have an internal only test where we want to substitute Xee for computePixels. Does that test pass with this change?

Comment (alxmrs) [reply xee/ext.py:641]: I think @shoyer would be good to consult, too.

Comment (dabhicusp) [reply xee/ext.py:641]: No @alxmrs sai's modulus trick is not working on all the images.
And Yes, that internal test was passed with this change.

Comment (alxmrs) [reply xee/ext.py:641]: Then this sounds good to me!

**Review by alxmrs — APPROVED** [review]
LGTM. Can you get the checks to pass?

**Comment by dabhicusp** [comment]
Yes @alxmrs all the CI/CD test-cases were passed locally and once this PR will merged I will update the internal test-case too.

**Pull request merged**